pkg/resourcecreator/istio.go

```diff
@@ -68,6 +64,17 @@ func ServiceRoleBinding(app *nais.Application) (*istio_crd.ServiceRoleBinding, e
 		})
 	}
 
+	if app.Spec.Prometheus.Enabled {
+		rules = append(rules, nais.AccessPolicyGressRule{
+			Namespace:   "istio-system",
+			Application: "default",
```

Comment:
Curious why the application is named "default"?

Reply:
Bare for å opprette binding til `cluster.local/ns/istio-system/sa/default`.

Reply:
Dunno why it's called default, but that is what's used in the istio prometheus installation:
https://github.com/istio/istio/blob/d8b9a10661c1d1ded591cbef39303c83fb1ed56f/install/kubernetes/helm/istio/charts/prometheus/templates/deployment.yaml#L70